app/models/foreman_virt_who_configure/output_generator.rb

```diff
@@ -125,17 +125,19 @@ def virt_who_output(format = nil)
   if [ $? -ne 0 ]; then result_code=$(($result_code|#{error_code(:virt_who_config_file_issue)})); fi
 
   step 4 "Creating sysconfig virt-who configuration"
-  cat > #{sysconfig_file_path} << EOF
+  cat > #{default_config_path} << EOF
 ### This configuration file is managed via the virt-who configure plugin
 ### manual edits will be deleted.
+[global]
 VIRTWHO_DEBUG=#{config.debug? ? 1 : 0}
```

Comment:
I wonder if the case of these config entries matters in newer versions like the one Will was referring to? ie ONESHOT vs oneshot